pypeit/core/pydl.py

```diff
@@ -652,7 +653,11 @@ def value(self, x, x2=None, action=None, lower=None, upper=None):
         if self.npoly > 1:
             goodcoeff = self.coeff[:, coeffbk]
         else:
-            goodcoeff = self.coeff[coeffbk]
+            try:
```

Comment:
remove try block?

Reply:
Same point as above. Will clean it up before we merge.